CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# 2.0.0
+
+Notes on migration of 1.1.0 data:
+
+```
+$ docker-compose exec database /bin/sh
+# mongo
+# use rsd
+```
+
+Delete field ``subtitle`` from all ``project`` documents:
```

Comment:
Due to alternating between code blocks and Markdown it is hard to execute all of migrations steps.

Please make a single large code block like so it can be run in one go:

```
// Delete field ``subtitle`` from all ``project`` documents
db.project.update({}, {$unset: {"subtitle": ""}}, {"multi": true})
// Delete field ``principalInvestigator`` from all ``project`` documents
db.project.update({}, {$unset: {"principalInvestigator": ""}}, {"multi": true})
```